contribute/developer_guide/README.md

```diff
@@ -43,25 +43,47 @@ scaffolded files consist of placeholders which can then be filled as desired.
   $ cat attributes.yaml 
```

Comment:
the repository part should point to the litmuschaos/litmus repo. keep the example with the pod_delete to make it clear

Reply:
I expect something like: To start automatically generating a new chart, edit the following file ... The above files will be generated automatically in the experiment folder.